### PR TITLE
Added a new query as FAQ

### PR DESCRIPTION
**Pull request by joinimran**
As per suggestions, added a new point in FAQs section to guide users if they want to show the computer name in diagnostic data.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4740

### CODE REVIEW

File: windows/deployment/update/device-health-using.md [hunk 1]
```diff
@@ -194,7 +194,8 @@ For example:
 
 *DHAppReliability | where AppFileDisplayName == "Microsoft Outlook"*
 
-
+#### Why does computer name shown as Unknow?
```
Comment (illfated) [inline windows/deployment/update/device-health-using.md]:
```suggestion
#### Why does the computer name show up as Unknown?
```
(alternative: Why is the computer name shown as Unknown?)

File: windows/deployment/update/device-health-using.md [hunk 2]
```diff
@@ -194,7 +194,8 @@ For example:
...
+To enhance consumer privacy, recent versions of Windows 10 no longer include the device name in Windows Diagnostic Data by default. To enable the devices to send their device name, you need to configure ***AllowDeviceNameInTelemetry*** via CSP or Group Policy.
```
Comment (illfated) [inline windows/deployment/update/device-health-using.md]:
```suggestion
To improve consumer privacy, recent versions of Windows 10 no longer include the device name in Windows Diagnostic Data by default. To enable the devices to send their device name, you need to configure ***AllowDeviceNameInTelemetry*** via CSP or Group Policy.
```
(This is more about a nuance in choice of words than anything actually being incorrect.)

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review done by illfated. Thank you.

**Comment by joinimran** [comment]
Suggestions applied, Ready to go live.

File: windows/deployment/update/device-health-using.md (hunk 1 above)
Comment (mypil) [reply windows/deployment/update/device-health-using.md]: @joinimran - please merge this commit on a suggested copy/edit review. Thanks.

**Comment by mypil** [comment]
@illfated - Thanks for helping do the copy/edit review of this PR.

**Comment by mypil** [comment]
@jaimeo - PR has been copy edited and is ready for final review. This is based on issue #4740.

Thank you.

**Comment by jaimeo** [comment]
@MatthewMWR  is this edit OK with you? Do not merge w/o this confirmation from the PM. 

Some changes:

- Delete "consumer."
- "Recent versions" will not fly--specify starting with which version of Windows 10 this is the case.

**Comment by jaimeo** [comment]
Actually, let's do this: This question was already answered in the Troubleshooting topic. If you insist on duplicating the information in this topic, just copy/paste the same text from https://docs.microsoft.com/en-us/windows/deployment/update/windows-analytics-faq-troubleshooting#device-names-not-appearing-for-windows-10-devices:


Device names not appearing for Windows 10 devices

Starting with Windows 10, version 1803, the device name is no longer collected by default and requires a separate opt-in. For more information, see Enrolling devices in Windows Analytics. Allowing device names to be collected can make it easier for you to identify individual devices that report problems. Without the device name, Windows Analytics can only label devices by a GUID that it generates.

**Comment by mypil** [comment]
@joinimran - please apply the recommended changes to this PR from jaimeo. Thank you.

**Comment by joinimran** [comment]
Suggestions applied. Ready for review.

**Comment by jaimeo** [comment]
Thanks, joinimran!

**Pull request merged**